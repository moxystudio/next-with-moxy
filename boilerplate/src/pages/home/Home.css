@import "../../shared/styles/variables";

.home {
    width: var(--home-width);
    height: var(--home-height);
    background-color: var(--color-white);
}
