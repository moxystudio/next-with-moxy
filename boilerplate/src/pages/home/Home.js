import React from 'react';
import { Contacts } from '../../shared/components';

import styles from './Home.css';

const Home = () => (
    <div className={ styles.home }>
        Home Page
        <Contacts />
    </div>
);

export default Home;
