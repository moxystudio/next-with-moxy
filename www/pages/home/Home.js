import React from 'react';
import { FormattedMessage } from 'react-intl';

import styles from './Home.module.css';

const Home = () => (
    <div className={ styles.home }>
        <h1><FormattedMessage id="home.title" /></h1>
    </div>
);

export default Home;
