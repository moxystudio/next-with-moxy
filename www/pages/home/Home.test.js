import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

it('should render correctly', () => {
    const tree = shallow(<Home />);

    expect(tree).toMatchSnapshot();
});
