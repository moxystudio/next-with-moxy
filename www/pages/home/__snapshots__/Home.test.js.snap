// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="home"
>
  Home Page
  <Contacts />
</div>
`;
